components/kyma-environment-broker/internal/process/provisioning/simple_provision.go

```diff
@@ -0,0 +1,111 @@
+package provisioning
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/process"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/servicemanager"
+	"github.com/sirupsen/logrus"
+)
+
+type overrideInputFunc func(*servicemanager.ProvisioningInput) *servicemanager.ProvisioningInput
+type infoExtractor func(*internal.ProvisioningOperation) *internal.ServiceManagerInstanceInfo
+
+type SimpleProvisioner struct {
```

Comment:
It is about Service Manager so I'd like to have a name which corresponds with SM, for example ServiceManagerProvisioner